examples/squeezer_bin/squeezer_bin/mod.rs

```diff
@@ -0,0 +1,45 @@
+mod imp;
+
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+
+glib::wrapper! {
+    pub struct SqueezerBin(ObjectSubclass<imp::SqueezerBin>) @extends gtk::Widget;
+}
+
+impl SqueezerBin {
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> Self {
+        glib::Object::new(&[]).unwrap()
+    }
+
+    pub fn keep_aspect_ratio(&self) -> bool {
+        self.imp().keep_aspect_ratio.get()
+    }
+
+    pub fn set_keep_aspect_ratio(&self, keep_aspect_ratio: bool) {
+        self.imp().keep_aspect_ratio.set(keep_aspect_ratio);
```

Comment:
You should check whether the property changed and then notify.
Something like
```rust
let imp =  self.imp();
if imp.keep_aspect_ratio.get() == keep_aspect_ratio {
   return;
} 

// /do what you are doing now
self.notify("keep-aspect-ratio"); // this is needed because you have explicit notify.

Reply:
Should be done too, but not sure